### PR TITLE
nested lists support and python3 compatibility

### PR DESCRIPTION
**Pull request by Maunty**

### CODE REVIEW

File: bareos/files/bareos-config.jinja [hunk 1]
```diff
@@ -16,23 +16,35 @@
...
+        {%- for var in y %}
+                {%- if var is mapping %}
+{{ x|indent(ind1, True) }} {
+{{ _config(var, ind1+ind_increment) }}
+{{ '}'|indent(ind1, True) }}
+                {%- elif var is iterable and var is not string %}
+{{ _list(x,var, ind1) }}            
+                {%- else %}
+{{ x|indent(ind1, True) }}{{ operator }}{{ var }}
+        {%- endif %}
```
Comment (javierbertoli) [inline bareos/files/bareos-config.jinja]: A silly detail, but mind you indent in this `{% endif %}` to match the rest of the block?

**Comment by Maunty** [comment]
done :) sorry for typo in commit message :)

**Comment by Maunty** [comment]
w8 a little bit, i saw in my tests something that i didnt mentioned.
This PR adds work with dict nested in list, but nested lists gives not what i expected.

**Comment by Maunty** [comment]
Nope. Its all right.
I was confused because i wanted to recive this kind of config:
```
FileSet {
    Name = "some_name"
    Include {
        File = "some_dir"
        Options {
            Signature = MD5
        }
    }
    Include {
        Options {
            wilddir = "some_regex"
            wildfile = "some_regex"
        }
        Options {
            Exclude = yes
            RegexFile = "some_regex"
        }
        File = "some_dir"
    }
```

and problem was in yaml, not im jinja. Valid yaml is : 
```yaml
director:
    config:
      FileSet:
        some_name:               
          Include:
            - Options:
                Signature: MD5
              File: '"some_dir"'
            - Options:
                  - wildfile: '"some_regex"'
                    wilddir: '"some_regex"'
                  - Exclude: 'yes'
                    RegexFile: '"some_regex"'
               File: '"some_dir"'
```

**Comment by javierbertoli** [comment]
Thanks, @Maunty !

**Pull request merged**